examples/animation/main.rs

```diff
@@ -13,30 +13,52 @@ use amethyst::renderer::{AmbientColor, Camera, DisplayConfig, DrawShaded, Elemen
                          KeyboardInput, Light, Mesh, Pipeline, PointLight, PosNormTex, Projection,
                          RenderBundle, Rgba, Stage, VirtualKeyCode, WindowEvent};
 use amethyst_animation::{get_animation_set, Animation, AnimationBundle, AnimationCommand,
-                         EndControl, InterpolationFunction, Sampler, StepDirection,
-                         TransformChannel};
+                         AnimationSet, DeferStartRelation, EndControl, InterpolationFunction,
+                         Sampler, SamplerPrimitive, StepDirection, TransformChannel};
 use genmesh::{MapToVertices, Triangulate, Vertices};
 use genmesh::generators::SphereUV;
 
-const SPHERE_COLOUR: [f32; 4] = [0.0, 0.0, 1.0, 1.0]; // blue
-const AMBIENT_LIGHT_COLOUR: Rgba = Rgba(0.01, 0.01, 0.01, 1.0); // near-black
-const POINT_LIGHT_COLOUR: Rgba = Rgba(1.0, 1.0, 1.0, 1.0); // white
-const BACKGROUND_COLOUR: [f32; 4] = [0.0, 0.0, 0.0, 0.0]; // black
+const SPHERE_COLOUR: [f32; 4] = [0.0, 0.0, 1.0, 1.0];
+// blue
```

Comment:
Comments should be on top, not after

Reply:
rustfmt ftl

Reply:
they used to be after the line, it moved it down ..

Reply:
Done